src/Angular.js

```diff
@@ -888,7 +889,59 @@ var csp = function() {
   return (csp.isActive_ = active);
 };
 
+/**
+ * @ngdoc directive
+ * @module ng
+ * @name ngJq
+ *
+ * @element ANY
+ * @param {string=} the name of the library available under `window`
+ * to be used for angular.element
+ * @description
+ * Use this directive to force the angular.element library.  This should be
+ * used to force either jqLite by leaving ng-jq blank or setting the name of
+ * the jquery variable under window (eg. jQuery).
+ *
+ * Since this directive is global for the angular library, it is recommended
+ * that it's added to the same element as ng-app or the HTML element, but it is not mandatory.
+ *
+ * @example
+ * This example shows how to force jqLite using the `ngJq` directive to the `html` tag.
+ ```html
+ <!doctype html>
+ <html ng-app ng-jq>
+ ...
+ ...
+ </html>
+ ```
+ * @example
+ * This example shows how to use a jQuery based library of a different name.
+ * The library name must be available at the top most 'window'.
+ ```html
+ <!doctype html>
+ <html ng-app ng-jq="jQueryLib">
+ ...
+ ...
+ </html>
+ ```
+ */
+var jq = function() {
+  if (isDefined(jq.name_)) return jq.name_;
+	var el;
+	var i, ii = ngAttrPrefixes.length;
+	for (i = 0; i < ii; ++i) {
+		if (el = document.querySelector('[' + ngAttrPrefixes[i].replace(':', '\\:') + 'jq]')) {
+			break;
+		}
+	}
+
+  var name;
+  if (el) {
+    name = getNgAttribute(el, "jq") || '';
```

Comment:
```js
var name;
if (el) {
  name = getNgAttribute(el, "jq");
}
```
This would result in `name` being `undefined` if there is no `ngJq` and `null` if `ngJq` is set but empty.

Then on [line 1516][1]: 

```js
var jqName = jq();
jQuery = isUndefined(jqName) ? window.jQuery :   // No `ngJq`; use default jQuery.
    ((jqName === null) ? undefined :             // `ngJq` present, but empty; use jqLite.
                         window[jqName]);        // Use jQuery specified by `ngJq`.
```

[1]: https://github.com/angular/angular.js/pull/10761/files#diff-1d54c5f722aebc473dbe96f836ddf974R1516